### ISSUE TITLE
TileSet zooming/positioning of one spritesheet changes zooming/positioning in others in all TileMaps

### ISSUE THREAD

**Issue by wyattbiker** [godotengine/godot#88150]
### Tested versions

v4.2.1.stable.official [b09f793f5]

### System information

Linux Kubuntu 23.04 KDE Plasma Version: 5.27.4 KDE Frameworks Version: 5.104.0 Qt Version: 5.15.8

### Issue description

When editing TileSets, zooming/positioning a spritesheet changes the other ones in all TileMaps to the same level. It is an annoyance and impediment especially dealig with different sized sheets to have to go back reposition and rezoom. Each tilesheet in a tilemap should keep its zoom level and position. see issue recording below

With one TileMap

https://github.com/godotengine/godot/assets/930478/e4e0dee1-8f95-4cc2-ba0e-610a67f43942

2 TileMaps

https://github.com/godotengine/godot/assets/930478/2cfee809-14ed-4fa4-81b3-9a771391b8ba

### Steps to reproduce

see screen recording

### Minimal reproduction project (MRP)

see screen recording

**Comment by Calinou** [comment]
I agree each TileSet should have its own zoom level, although this may prove to be an annoyance if we don't default to a suitable zoom level when opening new TileSets for the first time;

**Comment by wyattbiker** [comment]
Well currently it opens up at 100%, center top. I don't see why that needs to change, but I guess it could be saved as part of the tileset.
